### ISSUE TITLE
Loss only on frontier?

### ISSUE THREAD

**Issue by leyuan-sun** [srama2512/PONI#30]
Hi, still me. 
I have question about the MSE loss calculation, in the paper you mantioned that the loss only calculate on the frontier pixels, but I check the 
 ```
mask = labels["loss_masks"][:, 2:]
 # Evaluate predictions only on mask = 1 regions
 mask_sum = mask.sum(dim=3).sum(dim=2) + 1e-16  # (b, 1)
``` 
The mask is a whole floor map not the frontier pixel by visualization, so you calaucte on the entier floor map instead of frontier pixels, am I correct?

Besides, why there is no mask processing on area_pf loss?
```
area_pf_loss = self.area_loss_fn(area_preds, area_gts).mean()
loss = loss + area_pf_loss
losses["area_pf_loss"] = area_pf_loss.item()
```
